From e61ab761ab3858cd20e8350ed4384cede4b570e8 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Thu, 10 Dec 2015 11:04:35 +0100
Subject: [PATCH] MSys: Make sure that setmode() is declared

In MSys2's context, `_setmode()` is declared, and `setmode()` will be
defined for legacy only when including io.h. So let's do that.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 main.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/main.c b/main.c
index e318cbc..c3bd958 100644
--- a/main.c
+++ b/main.c
@@ -31,6 +31,10 @@ extern int do_getch();
 #endif				/* defined(USE_GPM) || defined(USE_SYSMOUSE) */
 #endif
 
+#ifdef __MSYS__
+#include <io.h>
+#endif
+
 #ifdef __MINGW32_VERSION
 #include <winsock.h>
 
-- 
2.6.3

